dnf5/commands/check-upgrade/check-upgrade.cpp

```diff
@@ -153,8 +176,13 @@ void CheckUpgradeCommand::run() {
             std::move(advisories.value()), installed_query, libdnf5::sack::QueryCmp::GTE);
     }
 
-    // Last, only include latest upgrades
-    upgrades_query.filter_latest_evr();
+    if (minimal) {
```

Comment:
I believe this should be `minimal->get_value()`.
It is also breaking a couple of tests.

Reply:
May I ask you to explain your comment in more details?

Reply:
There is an update for tests - https://github.com/rpm-software-management/ci-dnf-stack/pull/1511

Reply:
I've got it, the test was really incorrect. The test was for pointer and not for bool value in the pointer of BoolOption. Fixed